homeassistant/components/bthome/binary_sensor.py

```diff
@@ -0,0 +1,98 @@
+"""Support for BTHome binary sensors."""
+from __future__ import annotations
+
+from typing import Optional
+
+from bthome_ble import (
+    BTHOME_BINARY_SENSORS,
+    HOME_ASSISTANT_BINARY_SENSOR_DEVICE_CLASSES,
+    SensorUpdate,
+)
+
+from homeassistant import config_entries
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .device import device_key_to_bluetooth_entity_key, sensor_device_info_to_hass
+
+BINARY_SENSOR_DESCRIPTIONS = {}
+for key in BTHOME_BINARY_SENSORS:
+    # Not all BTHome device classes are available in Home Assistant
+    DEV_CLASS = None
+    if key in HOME_ASSISTANT_BINARY_SENSOR_DEVICE_CLASSES:
```

Comment:
Instead of maintaining such a list, do this:

```python
from homeassistant.components.binary_sensor import BinarySensorDeviceClass

try:
    BinarySensorDeviceClass(key)
except ValueError:
    key = None
```

Reply:
Changed it, however, I can't use key = None, as the key of one of the binary sensors is "generic", only the device class should be `None`